### ISSUE TITLE
Set the Profile and Team For Targets automatically

### ISSUE THREAD

**Issue by tringManoj** [fastlane/fastlane#8621]
Hi,
I can fetch the latest Profile for the targets via MATCH (Ad-Hoc & App Store). When I open Xcode build setting I see them all there. But the problem is for few targets I see it's not set automatically. It appears in red with red colour text. So I need to set in manually. Is there a way where it can automatically set the profile. So there wont be any manual work to set the profile for each after fetching the profiles via MATCH? 

### New Issue Checklist

- [ ] Updated fastlane to the latest version
- [ ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

[INSERT OUTPUT HERE]

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

[INSERT OUTPUT HERE]

**Comment by mgrebenets** [comment]
Hi @tringManoj 

Sounds like you are describing issue with Xcode itself.
Match would pull the profiles and by default put them to `~/Library/MobileDevice/Provisioning Profiles`. Then it would be up to Xcode to automatically pick profile based on target's bundle ID. But that's Xcode... so we all know how good it is with automatic things.

To me personally it sounds like a one-off action - you'd have to fix it only once for each new target created. Match also doesn't touch Xcode project build settings. Doing that is not what match is designed for, it seems.

Sounds like you could have a plugin to automate this task, it if really becomes too much to set it by hand. Then plugin, in the long run, could possibly be moved into core fastlane tools, if it's proven to be valuable enough.

**Comment by PetrosIchor** [comment]
Just to add to the comment above, there's a number of plugins in the community for injecting the xcode configuration needed when signing.  Sounds like you're looking to mail the profile values to your target at build time. Check out actions/plugins around modifying those project files.
